extensions/amp-mustache/0.2/amp-mustache.js

```diff
@@ -86,6 +92,30 @@ export class AmpMustache extends AMP.BaseTemplate {
     });
   }
 
+  /**
+   * Stores and replaces nested script templates with custom triple-mustach
+   * pointers.
+   *
+   * This prevents the outer-most script template from replacing variables in
+   * nested templates. Note that this constrains nested template markup to the
+   * more restrictive sanitization rules of triple-mustache.
+   *
+   * @param {!Element} content
+   */
+  processNestedScriptTemplates_(content) {
```

Comment:
Why are we duplicating?

Reply:
Initially, what was written v0.2 was already the same logic to what was in amp-mustache v0.1 and inlined and not just extracted into its own method. I just pulled it out to its own method. I will pull it out and the template versions can share the logic once I add support for nested templates in the next PR.

Reply:
Hm, it's the diff that's screwed up. It's not showing me the old code.